workbench-core/authorization/src/dynamicAuthorization/groupStatus.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Enum of the status of a group
+ */
+export enum GroupStatus {
```

Comment:
Doesn't need to be a enum, use types instead.

Reply:
changed.